src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -469,6 +469,12 @@ export class ResultsViewPageStore {
 
     @observable queryFormVisible: boolean = false;
 
+    @observable public selectedMolecularProfileNameForDownload: string | undefined = undefined;
+
+    @computed get isOtherMolecularProfileExist() {
```

Comment:
could you please explain what this is? I'm confused by the name

Reply:
@adamabeshouse this is an indicator for the `nonSelectedMolecularProfiles`. This means if there is a molecular profile in the study and the user doesn't select this/those profiles. So maybe should be called `isNonSelectedMolecularProfilesExist`?

Reply:
ah okay, yeah I like your suggestion much better. the right grammar is `doNonSelectedMolecularProfilesExist`

Reply:
@adamabeshouse Thanks!